include/aws/common/process.h

```diff
@@ -21,6 +21,8 @@ AWS_EXTERN_C_BEGIN
 
 AWS_COMMON_API int aws_get_pid(void);
 
+AWS_COMMON_API struct aws_string *aws_get_command_output(struct aws_allocator *allocator, const char *command);
```

Comment:
I think something like aws_run_command() or similar would be a better name.  Getting the output is reasonably expected; the real action is running something.  Also, I'd do the standard int return and the output as an output parameter